.eslintrc

```diff
@@ -0,0 +1,33 @@
+{
+  "parser": "babel-eslint",
```

Comment:
For the ESLint, Prettier  we should use the same rules  we use here: https://gitlab.com/SUSE-UIUX/eos-user-story/-/blob/master/.eslintrc.yml for consistency

Reply:
Okay I'll do the changes